loader/loader.lua

```diff
@@ -55,7 +55,7 @@ function loadMods(modsDirectory)
         },
         prefix        = { pattern = '%-%-%- PREFIX: (.-)\n' },
         version       = { pattern = '%-%-%- VERSION: (.-)\n', handle = function(x) return x and V(x):is_valid() and x or '0.0.0' end },
-        outdated      = { pattern = { 'SMODS%.INIT', 'SMODS%.Deck' } },
+        outdated      = { pattern = { 'SMODS%.INIT', 'SMODS.Deck[:.]new' } },
```

Comment:
the %. is nessicary as . represents any character, and the % escapes it

Reply:
Like this?